### PR TITLE
feat: add basic form validation for client

### PR DESCRIPTION
**Pull request by Akirtovskis**
- Setup yup schema for digital ocean access token and server name
- Min and max checks were added to give more feedback if token has not pasted properly or something extra entered
- Added helper text in UI component
- Nice addition to this would be disabled buttons, perhaps todo for @bartaxyz ?

Screenshot 2020-05-03 at 10 34 38

### CODE REVIEW

File: client/src/pages/onboarding/cloud-provider.tsx [hunk 1]
```diff
@@ -27,11 +27,12 @@ const CloudProvider = () => {
...
   ] = useSaveDigitalOceanAccessTokenMutation();
-  const digitalOceanAccessTokenSchema = Yup.object().shape({
-    token: Yup.string()
+  const digitalOceanAccessTokenSchema = yup.object().shape({
+    token: yup
+      .string()
       .trim()
-      .min(64, 'Token too short, it should be exactly 64 chars long')
-      .max(64, 'Token too long, it should be exactly 64 chars long')
+      .min(64, 'Token too short, it should be exactly 64 characters long')
```
Comment (pradel) [inline client/src/pages/onboarding/cloud-provider.tsx:34]: @Akirtovskis the regex already say that the string should be 64 chars long, so is this really needed?

Comment (Akirtovskis) [reply client/src/pages/onboarding/cloud-provider.tsx:34]: @pradel idea behind this was to give better feedback, when for example you paste incomplete token or enter something afterwards, but could be also removed. What do you think?

**Comment by Akirtovskis** [comment]
@pradel server side validation added 🚀

File: client/src/pages/onboarding/cloud-provider.tsx (hunk 1 above)
Comment (pradel) [reply client/src/pages/onboarding/cloud-provider.tsx:34]: Yeah make sense 👍

**Comment by pradel** [comment]
@Akirtovskis 🚀

**Pull request merged**